users/dlip/taipo.h

```diff
@@ -0,0 +1,540 @@
+// Copyright 2023 Dane Lipscombe (@dlip)
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Special thanks to Whorf for his help with this
+
+#pragma once
+#include "keycodes.h"
+#include "quantum_keycodes.h"
+#define IDLE_TIMEOUT_MS 150
+#define TAIPO_KEYCODE_OFFSET (SAFE_RANGE + 500)
```

Comment:
Indexing like this is invalid. As it stands it overflows into the unicode address space.